### PR TITLE
chore(heureka): renames Issues to Vulnerabilities tab with the default filter of issueType is Vulnerability

### PR DESCRIPTION
**Pull request by hodanoori**
# Summary

This pull request introduces a default filter of `issueType=Vulnerability `to the `issueMatches` entity, ensuring that only vulnerabilities associated with various images are displayed on this view. Additionally, related text updates have been implemented for consistency and clarity.

# Changes Made

- **Tab Update:** The "Issues" tab has been renamed to "Vulnerabilities" for better alignment with the content displayed.
- **Detail Panel Simplification:** The issueType field has been removed from the details panel since the view now applies a default `issueType=Vulnerability` filter.
- **Text Adjustments:** "Not found" messages have been updated to reference "vulnerabilities" instead of "issues" and "images" instead of "components" to improve clarity.
- **Filter Option Adjustment:** The `issueType` filter option has been removed from the proposed filter select list, reflecting the default filtering logic applied.
- **Function Enhancement:** The `setQueryOption` function has been modified to merge new queryOptions with the existing ones rather than replacing them entirely.

# Related Issues

- Issue 1: https://github.com/cloudoperators/heureka/issues/418

# Checklist

- [ ] I have performed a self-review of my code.
- [ ] I have commented my code, particularly in hard-to-understand areas.
- [ ] I have added tests that prove my fix is effective or that my feature works.
- [ ] New and existing unit tests pass locally with my changes.
- [ ] I have made corresponding changes to the documentation (if applicable).
- [ ] My changes generate no new warnings or errors.

### CODE REVIEW

**Review by barsukov — COMMENTED** [review]
I left one comment as a question

File: apps/heureka/src/components/issueMatches/IssueMatchesView.jsx [hunk 1]
```diff
@@ -21,6 +22,12 @@ const IssueMatchesView = () => {
   const labels = useIssueMatchesFilterLabels()
   const filterLabelValues = useIssueMatchesFilterLabelValues()
   const searchTerm = useIssueMatchesSearchTerm()
+
+  // Access filter actions
+  const { setQueryOptions } = useGlobalsActions()
+
+  setQueryOptions("IssueMatches", { filter: { issueType: ["Vulnerability"] } })
```
[the remark this answers is not in the document]
Comment (hodanoori) [reply apps/heureka/src/components/issueMatches/IssueMatchesView.jsx:29]: yes

**Review by barsukov — APPROVED** [review]
LGTM please ask someone to review it proper

**Review by TilmanHaupt — APPROVED** [review]
looks logical to me!

**Pull request merged**